src/icons/fileIcons.ts

```diff
@@ -1094,6 +1094,7 @@ export const fileIcons: FileIcons = {
         '.firebaserc',
         'firestore.rules',
         'firestore.indexes.json',
+        'storage.rules',
```

Comment:
I think this one is too generic. That means that there might be other projects outside that are not related to Firebase but still use files like "storage.rules". If we then show the Firebase icon there it might be a bit confusing. However, you can configure this in your VS Code user settings:

```json
"material-icon-theme.files.associations": {
    "storage.rules": "firebase"
}
```